dev/com.ibm.ws.security.acme_fat/fat/src/com/ibm/ws/security/acme/docker/PebbleContainer.java

```diff
@@ -33,13 +33,7 @@
 /**
  * Testcontainer implementation for the letsencrypt/pebble container.
  */
-public class PebbleContainer extends GenericContainer<PebbleContainer> {
-
-	/**
-	 * The port that ACME HTTP validation requests will be sent to the domain
-	 * on. Must match the port in 'publish/files/pebble-config.json'.
-	 */
-	public static final int HTTP_PORT = 5002;
+public class PebbleContainer extends CAContainer {
```

Comment:
Again, I expected a wrapper class to extend CAContainer and to contain the Pebble and Challtestserv containers.